src/AnomalyDetection/query.js

```diff
@@ -6,8 +6,11 @@ const timesliceParams = [
   'time_zone:America/New_York',
   'anomaly:true',
 ].join(',');
-const anomalyAgg = `timeslice(${timesliceParams}).top_hits(1)`;
-
+const anomalyParams = [
```

Comment:
probably would be better to do something like:
```
const anomalyAgg = [
  `timeslice(${timesliceParams})`,
  `term(${fields.text_keyword_text},count:1)`,
  `term(${fields.title},count:1)`
].join(',');
```
then add the `text_keyword_text` and `title` fields to the `fields.js` file